common/src/test/java/com/netflix/conductor/common/tasks/TaskTest.java

```diff
@@ -78,13 +78,15 @@ public void testTaskDefinitionIfAvailable() {
     @Test
     public void testTaskQueueWaitTime() {
         Task task = new Task();
-        task.setScheduledTime(System.currentTimeMillis() - 30_000); // 30 seconds ago
-        task.setStartTime(System.currentTimeMillis() - 25_000);
```

Comment:
The assertion at line 85 fails if `System.currentTimeMillis()` does not return the same value on lines 81 and 82.